scalafix-core/src/main/scala/scalafix/util/Severity.scala

```diff
@@ -0,0 +1,23 @@
+package scalafix.util
+
+import scalafix.config.ReaderUtil
+
+import metaconfig.Reader
+
+sealed abstract class Severity(color: String, val order: Int)(
+    implicit name: sourcecode.Name)
+    extends Ordered[Severity] {
+  override val toString: String =
+    color + name.value.toLowerCase + Console.RESET
```

Comment:
Good catch. It should be possible to disable color through .scalafix.conf, I opened #120 to track this.